test/infinity.test.js

```diff
@@ -0,0 +1,47 @@
+'use strict'
+
+const t = require('tap')
+const test = t.test
+const proxyquire = require('proxyquire')
+const build = proxyquire('..', { long: null })
+
+test('render Infinity as null (integer)', (t) => {
+  t.plan(1)
+
+  const schema = {
+    title: 'integer',
+    type: 'integer'
+  }
+
+  const stringify = build(schema)
+  const output = stringify(Infinity)
+
+  t.equal(output, 'null')
```

Comment:
I think this should be `0`

For this schema:

```
{
  title: 'integer',
  type: 'integer'
}
```

the user wants only integer as output

For this one instead, user accepts `null` value also
```
{
  title: 'integer',
  nullable: true,
  type: 'integer'
}
```

Reply:
Sure, will do :)

Reply:
@Eomm I'm not sure it should be `0`. That has a very different meaning than `Infinity`. But I'm also not sure what to do here. The spec merely says:

https://tools.ietf.org/html/rfc8259#section-6
> Numeric values that cannot be represented in the grammar below (such
   as Infinity and NaN) are not permitted.

Reply:
You are right @jsumners 

I would just don't output the field (instead of zero) and output `null` when it is nullable.
I say so because when a JSON doesn't fit a schema, `fast-json-stringify` simply doesn't process it (like `foo: {type: 'string'}` and but then foo is an array)

Reply:
@Eomm  ok cool, then I will change it that way.
One question I have though, what's then the right output if I pass `{ type: 'integer' }` as schema and `Infinity` as value? `` or `undefined`?

Reply:
Edge cases 🎉 

The target is to serialize only a number (odd, but it is ok):

```js
schema = { type: 'integer' }
serialize = fjs(schema)
serialize(Infinity)
```

I would avoid undefined because this would break clients that do `JSON.parse(undefined)`
For this reason, I would say `''`

Do you agree @jsumners ?

Reply:
It should just be omitted or generate an error. The JSON Schema spec defers to the JSON spec — https://datatracker.ietf.org/doc/html/draft-handrews-json-schema-02#section-6.1

Therefore, `{ foo: Infinity }` is invalid.

Reply:
We do so when want to serialize a json, but when we try to serialize a number only?
It is doable for this module and I quite agree to throw a "CANNOT SERIALIZE" error.

It would be the first error thrown TBH

Reply:
It is clearly an error according to spec.